README.md

```diff
@@ -1,9 +1,86 @@
-# amqp-node
+# hato
 
-![CircleCI](https://img.shields.io/circleci/build/github/openrm/amqp-node)
-![Code Climate maintainability](https://img.shields.io/codeclimate/maintainability/openrm/amqp-node)
-![Code Climate coverage](https://img.shields.io/codeclimate/coverage/openrm/amqp-node)
+[![CircleCI](https://img.shields.io/circleci/build/github/openrm/hato)](https://app.circleci.com/pipelines/github/openrm/hato?branch=master)
+[![Code Climate maintainability](https://img.shields.io/codeclimate/maintainability/openrm/hato)](https://codeclimate.com/repos/5eb8c1314deafc018c000c2a/maintainability)
+[![Code Climate coverage](https://img.shields.io/codeclimate/coverage/openrm/hato)](https://codeclimate.com/repos/5eb8c1314deafc018c000c2a/test_coverage)
+[![GitHub](https://img.shields.io/github/license/openrm/hato)](https://github.com/openrm/hato/blob/master/LICENSE)
+
+A minamalist, customizeable AMQP framework
+
+## Installation
+```sh
+$ git clone https://github.com/openrm/hato.git
+```
+
+## About
+hato aims to simplify messaging without diminishing the flexibility and power in doing so.
+
+This library is highly customizeable. Plugins allow for a configuration that fits your project and for extension of the library to meet unique requirements.
+
+## Getting Started
+
+Include the library and plugins
+
+```js
+const { Client, plugins } = require('hato');
+```
+
+
+Construct a new client with the plugins of your choosing
+```js
+const client = new Client(BROKER_URL, {
+    plugins: [
+        new plugins.GracefulShutdown(),
+        new plugins.ConnectionRetry(),
+        new plugins.Duplex(),
+        new plugins.Encoding('json'),
+        new plugins.RPC(),
+        new plugins.Confirm(),
+        new plugins.Retry()
```

Comment:
We could emphasize that users can use strings instead.
```node
    plugins: [
        'gracefulShutdown', // register with default options
        'connectionRetry',
        'duplex',
        new plugins.Encoding('json') // instantiate when you need a detailed configuration
    ]
```
But please note that this interface has the highest possibility of being modified in the near future.

Reply:
We should also note that one can just initialize without `plugins` option to start by default plugins.
`logger` option can be mentioned as well.